README.md

```diff
@@ -28,7 +28,7 @@
   </p>
 
   <p>
-  Unofficial implementation of Fluent UI for <a href="https://flutter.dev">Flutter</a>. It's written based on the <a href="https://docs.microsoft.com/en-us/windows/uwp/design">official documentation</a>.
+  Unofficial implementation of Windows UI for <a href="https://flutter.dev">Flutter</a>. It's written based on the <a href="https://docs.microsoft.com/en-us/windows/uwp/design">official documentation</a>.
```

Comment:
Also this ? 

I just don't know if it's a global rename that escape you or if you really wanted to move from the term Fluent to Windows ?